src/collection-facets.ts

```diff
@@ -322,146 +325,109 @@ export class CollectionFacets extends LitElement {
         </h1>
         <div class="facet-group-content ${isOpen ? 'open' : ''}">
           ${this.getFacetTemplate(facetGroup)}
+          ${this.searchMoreFacetsLink(facetGroup)}
         </div>
       </div>
     `;
   }
 
   /**
-   * Generate the list template for each bucket in a facet group
+   * Generate the More... link button just below the facets group
+   *
+   * TODO: want to fire analytics?
```

Comment:
im thinking we should not overload this component w analytics mgmt. can we fire lit reactive callbacks when needed so that collection-browser can catch it and fire it from the top.